source/Octopus.Tentacle.Client/Capabilities/CapabilitiesResponseV2ExtensionMethods.cs

```diff
@@ -18,14 +16,14 @@ public static bool HasScriptServiceV2(this CapabilitiesResponseV2 capabilities)
             return capabilities.SupportedCapabilities.Contains(nameof(IScriptServiceV2));
         }
 
-        public static bool HasScriptServiceV3Alpha(this CapabilitiesResponseV2 capabilities)
+        public static bool HasAnyKubernetesScriptService(this CapabilitiesResponseV2 capabilities)
```

Comment:
Is it worth pulling this out into some sort of k8s directory owned by your team? So you don't need SaST approval for future changes to this?

Reply:
Yep, I can do that :)

Reply:
Moved in 0dff6b00